examples/with-scss/razzle.config.js

```diff
@@ -0,0 +1,73 @@
+// Based on: https://gist.github.com/jaredpalmer/0a91a7bd354b875b913c74f4b16125f7
+
+const path = require('path')
+const autoprefixer = require('autoprefixer')
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
+
+module.exports = {
+  modify: (baseConfig, { target, dev }, webpack) => {
+
+    const appConfig = Object.assign({}, baseConfig)
+    const isServer = target !== 'web'
+
+    const postCssLoader = {
+      loader: 'postcss-loader',
+      options: {
+        ident: 'postcss', // https://webpack.js.org/guides/migrating/#complex-options
+        plugins: () => [
+          autoprefixer({
+            browsers: [
+              '>1%',
+              'last 4 versions',
+              'Firefox ESR',
+              'not ie < 9' // React doesn't support IE8 anyway
+            ]
+          })
+        ]
+      }
+    }
+
+    appConfig.module.rules.push({
+      test: /.scss$/,
+      use:
+      // On the server, we simply use css-loader to deal with scss imports
+      isServer ? 'css-loader' :
```

Comment:
Replacing this line by `isServer ? ['css-loader', 'sass-loader'] :` seems to fix the `@import` not working in dev as commented there https://github.com/jaredpalmer/razzle/pull/314/#issuecomment-323797303

Reply:
Yep, this is what I'm doing in my app as well.

https://github.com/neoverse/explorer/blob/4aac1fe9eebb539d067cbf490ee7ee8a41cb6fd0/config/razzle-sass.js#L26

Reply:
Sure, I updated this to use both `css-loader` and `sass-loader` on the server.